app.js

```diff
@@ -2055,110 +2068,29 @@ App.RedirectRoute = Ember.Route.extend({
\\n    \"],[8],[0,\"\\n\\n    \"],[6,\"div\"],[9,\"class\",\"row open-source-previews\"],[7],[0,\"\\n      \"],[6,\"a\"],[9,\"href\",\"https://opensource.addepar.com/ember-table\"],[7],[0,\"\\n        \"],[6,\"div\"],[9,\"class\",\"col-md-4 preview-box\"],[7],[0,\"\\n          \"],[6,\"h3\"],[7],[0,\"Ember Table\"],[8],[0,\"\\n          \"],[6,\"img\"],[9,\"src\",\"/img/preview_table.png\"],[7],[8],[0,\"\\n          \"],[6,\"p\"],[9,\"class\",\"reduced center-align\"],[7],[0,\"\\n            \"],[6,\"a\"],[9,\"target\",\"_BLANK\"],[9,\"href\",\"https://github.com/addepar/ember-table\"],[7],[6,\"i\"],[9,\"class\",\"icon-github\"],[7],[8],[0,\" View on GitHub\"],[8],[0,\"\\n            \"],[6,\"iframe\"],[9,\"src\",\"https://ghbtns.com/github-btn.html?user=addepar&repo=ember-table&type=watch&count=true\"],[9,\"allowtransparency\",\"true\"],[9,\"frameborder\",\"0\"],[9,\"scrolling\",\"0\"],[9,\"width\",\"110\"],[9,\"height\",\"20\"],[7],[8],[0,\"\\n          \"],[8],[0,\"\\n        \"],[8],[0,\"\\n      \"],[8],[0,\"\\n      \"],[6,\"a\"],[9,\"href\",\"https://opensource.addepar.com/ember-widgets\"],[7],[0,\"\\n        \"],[6,\"div\"],[9,\"class\",\"col-md-4 preview-box\"],[7],[0,\"\\n          \"],[6,\"h3\"],[7],[0,\"Ember Widgets\"],[8],[0,\"\\n          \"],[6,\"img\"],[9,\"src\",\"/img/preview_widgets.png\"],[7],[8],[0,\"\\n          \"],[6,\"p\"],[9,\"class\",\"reduced center-align\"],[7],[0,\"\\n            \"],[6,\"a\"],[9,\"target\",\"_BLANK\"],[9,\"href\",\"https://github.com/addepar/ember-widgets\"],[7],[6,\"i\"],[9,\"class\",\"icon-github\"],[7],[8],[0,\" View on GitHub\"],[8],[0,\"\\n            \"],[6,\"iframe\"],[9,\"src\",\"https://ghbtns.com/github-btn.html?user=addepar&repo=ember-widgets&type=watch&count=true\"],[9,\"allowtransparency\",\"true\"],[9,\"frameborder\",\"0\"],[9,\"scrolling\",\"0\"],[9,\"width\",\"110\"],[9,\"height\",\"20\"],[7],[8],[0,\"\\n          \"],[8],[0,\"\\n        \"],[8],[0,\"\\n      \"],[8],[0,\"\\n      \"],[6,\"a\"],[9,\"href\",\"https://opensource.addepar.com/ember-charts\"],[7],[0,\"\\n        \"],[6,\"div\"],[9,\"class\",\"col-md-4 preview-box\"],[7],[0,\"\\n          \"],[6,\"h3\"],[7],[0,\"Ember Charts\"],[8],[0,\"\\n          \"],[6,\"img\"],[9,\"src\",\"/img/preview_charts.png\"],[7],[8],[0,\"\\n          \"],[6,\"p\"],[9,\"class\",\"reduced center-align\"],[7],[0,\"\\n            \"],[6,\"a\"],[9,\"target\",\"_BLANK\"],[9,\"href\",\"https://github.com/addepar/ember-charts\"],[7],[6,\"i\"],[9,\"class\",\"icon-github\"],[7],[8],[0,\" View on GitHub\"],[8],[0,\"\\n            \"],[6,\"iframe\"],[9,\"src\",\"https://ghbtns.com/github-btn.html?user=addepar&repo=ember-charts&type=watch&count=true\"],[9,\"allowtransparency\",\"true\"],[9,\"frameborder\",\"0\"],[9,\"scrolling\",\"0\"],[9,\"width\",\"110\"],[9,\"height\",\"20\"],[7],[8],[0,\"\\n          \"],[8],[0,\"\\n        \"],[8],[0,\"\\n      \"],[8],[0,\"\\n    \"],[8],[0,\"\\n  \"],[8],[0,\"\\n\"],[8],[0,\"\\n\"]],\"hasEval\":false}","meta":{}});
```

Comment:
that's a very long line. can we split this up?

Reply:
Whitespaces cost money, bro!